com.georgefb.mangareader.yml

```diff
@@ -0,0 +1,26 @@
+app-id: com.georgefb.mangareader
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: mangareader
+rename-icon: mangareader
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
```

Comment:
what do you need host permissions for?

Reply:
Upon a closer look, it seems like your app operates on folders. Is there any tighter scoping possible?

Reply:
The host permission is required because the app extracts archives and the path where the archives are extracted if configurable.
> Is there any tighter scoping possible?
If you mean using another option for --filesystem then I don't think it's possible, as the user can add multiple folders to create a tree view and I don't want to tell them where their folders should be.